GOVERNANCE.md

```diff
@@ -1,45 +1,117 @@
 
-## Community Structure
+- evangelising about the project (e.g. a link on a website and word-of-mouth awareness raising)
+- informing developers of strengths and weaknesses from a new user perspective
+- providing moral support
+- providing financial support
 
-![community organization](/media/governance/community_organization.png)
+Users who continue to engage with the project and its community will often become more and more involved. Such users may find themselves becoming contributors, as described in the above section.
 
-The community structure comprises two parts. The first part is the community organization, and the second part is the role division within the community group and the promotion path of each community role.
+## Approving PRs
 
-PMC: The Project Management Committee; functions as the core management team that oversees the TiDB community. The PMC decides the roadmap of the project and makes major decisions related to the community.
+Unless otherwise requested by Maintainers, PRs may be merged after receiving at least two approvals from Reviewers or higer roles.
 
-Developer Group:
+## Decision making and voting
 
-- Maintainer: Mainatiners are the planners and designers of the project, with the authority to merge branches into the master. 
-- Committer:  Recommended by Maintainers or the PMC as Contributors with outstanding contributions to TiKV. A Committer must have completed at least one feature or fix a major bug independently.
-- Reviewer: Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Contributors. Reviewer has the rights to approve code reviews with LGTM (Looks Good To Me).
-- Active contributor: Contributed 8 PRs and above to TiKV within one year and received recognition from 2 incumbent Reviewers
-- Contributor: Merged at least 1 PR in any Repo under [TiKV](https://github.com/tikv) 
+Ideally, all project decisions are resolved by consensus via a PR or GitHub issue. If this is not possible, Maintainers may call a vote. Many of the day-to-day project maintenance can be done by a [lazy consensus](http://communitymgt.wikia.com/wiki/Lazy_consensus) model. But the following items must be called to vote:
 
-Organizer Committee:  The Organizer Committee consists of organizers in charge of event or activity operations; in charge of the execution of PMC’s strategies and decisions. Specifically, the members are the leaders of user groups or community event organizers across regions
+- Enforcing a Code of Conduct violation (super majority)
+- Adding or removing a maintainer/committer/SIG Leads (super majority)
+- Changing the governance rules in this document (super majority)
+- Licensing and intellectual property changes (including new logos, wordmarks) (simple majority)
+- Adding, archiving, or removing SIG/subproject (simple majority)
+- Other cases where consensus cannot be made (super majority)
```

Comment:
These are important things, but having a vote makes them easier to happen than reaching consensus, which I think is the opposite of what is desired.

Reply:
What do you suggest?

Reply:
Consensus for everything :-)

Reply:
So what are occasions for voting？